SPECS/k3s/k3s-1.23.8.signatures.json

```diff
@@ -0,0 +1,6 @@
+{
+    "Signatures": {
+     "k3s-1.23.8-vendor.tar.gz": "017636ddaf542442b9c8992fb8646a48a5d57efdecb4861c626e8ad919bbf905",
```

Comment:
This source does not seem to be available on the `blob-store`.
Can we please double check.

Reply:
It should be available now

Reply:
Sure, the signature does not seem to match to the one in the `blob-store` though.
I am getting: `3c96e864d0e89e318ecdd62e1750f787d1b622feeb240f7b86d9f3280447aeda` - Can you please double check again :)

Reply:
Updated. Thanks for pointing that out :)